control/control.rnc

```diff
@@ -189,6 +190,9 @@ displaymanager_shutdown =		element displaymanager_shutdown { STRING }
 ## Writes POLKIT_DEFAULT_PRIVS to /etc/sysconfig/security
 ## if set
 polkit_default_privs =			element polkit_default_privs { STRING }
+## Install prefered bootloader if it fits to the system
+## 'systemd-boot', 'grub2-efi' or 'grub2'
```

Comment:
I am not sure if it is good idea to allow both grub2-efi and grub2 as we use proper one for given architecture and way how iso is booted. So I would keep here only grub2 and systemd-boot with note that default is grub2.

Reply:
I have removed grub2-efi. Thanks !